components/javascript/toast.js

```diff
@@ -0,0 +1,23 @@
+const topToast = document.querySelector("#top-toast-id");
+const topToastContent = document.querySelector('#top-content');
+const topToastClose = document.querySelector('#top-toast-close');
+
+const bottomToast = document.querySelector('#bottom-toast-id');
+const bottomToastContent = document.querySelector('#bottom-content');
+const bottomToastClose = document.querySelector('#bottom-toast-close');
+
+const openTopToast = () => topToastContent.style.display = 'block';
+
+const closeTopToast = () => topToastContent.style.display = 'none';
```

Comment:
use `setTimeout` so that toast closes itself after a few seconds since this is the default behaviour of toast component

Reply:
Added setTimeout to close toast. Thank you for suggestion.